ledger/apply/application.go

```diff
@@ -199,6 +199,16 @@ func updateApplication(ac *transactions.ApplicationCallTxnFields, balances Balan
 	// Fill in the new programs
 	record.AppParams = cloneAppParams(record.AppParams)
 	params := record.AppParams[appIdx]
+	proto := balances.ConsensusParams()
+	// when proto.EnableExtraPageOnAppUpdate is false, WellFormed rejects all updates with a multiple-page program
+	if proto.EnableExtraPagesOnAppUpdate {
+		allowed := int(1+params.ExtraProgramPages) * proto.MaxAppProgramLen
```

Comment:
please check if this formula `int(1+params.ExtraProgramPages) * proto.MaxAppProgramLen` is used somewhere else in **ledger/apply** package, and if so, make a function.

Reply:
no other occurrence is found.